index[1].php

```diff
@@ -10,28 +20,46 @@
 		<meta name="author" content="Johno Devine">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="stylesheet" href="stylesheet.css"/>
+		<link rel="icon" type="image/x-icon" href="images/icon.ico">
 	</head>
 	<body>
+		<!-- banner for the website -->
 		<div class="banner"></div>
-		<div id="myNav" class="overlay">
-		  <a href="javascript:void(0)" class="closebtn" onclick="closeNav()"><span style="color: #ff244f;">&times;</span></a>
-		  <div class="overlay-content">
-			<a href="index.php">Home</a>
-			<a href="music.php">Music</a>
-			<a href="contact.php">Contact</a>
-		  </div>
-		</div>
-		<span style="font-size: 60px;cursor:pointer" onclick="openNav()">&#9776;</span>
-		
+		<!-- Checks if the user is Graeme or not -->
+		<?php
+		if(!isset($_SESSION['Form'])){
+			 ?>
+			<button class="btn" onclick="window.location.href='login.php';"> <img src="images/profile.png" class="profile_img" alt="profile image"/></button>
+			<?php
+		}
+		elseif ($_SESSION['Form'] != 'Graeme'){
+			?>
+			<button class="logout_btn" onclick="window.location.href='logout.php';"> Log Out </button>
```

Comment:
code that checks who is logged in and will display different things depending who is logged in (Admin or User)